Index.html

```diff
@@ -81,6 +113,124 @@ <h3 class="work-subheading">Multi-Post Stories</h3>
     </ul>
     <button type="button" onclick="" class="work-button">See Project</button>
 
+    <div class="desktop-project-grid"> 
+      <div class="desktop-project-1">
+        <img class="project-background" src="images/desktop-project-background.png" alt="Project Background Image"/>
+        <button type="button" onclick="" class="project-button">See Project</button>
+      </div>
+
+      <div class="desktop-project-2">
+        <img class="project-background" src="images/desktop-project-background-2.png" alt="Project Background Image"/>
+        <h2 class="desktop-project-title-2">Data Dashboard <br> Healthcare</h2>
+        <p class="desktop-project-description-2">
+          A daily selection of privately <br> 
+          personalized reads, no accounts or <br> 
+          sign-ups required has been <br> 
+          the industry's standard.
+        </p>
+        <ul class="project-list">
+          <li class="project-item">
+            html
+          </li>
+          <li class="project-item">
+            bootstrap
+          </li>
+          <li class="project-item">
+            Ruby
+          </li>
+        </ul>
+      </div>
+
+      <div class="desktop-project-3">
+        <img class="project-background" src="images/desktop-project-background-3.png" alt="Project Background Image"/>
+        <h2 class="desktop-project-title-3">Website Portfolio</h2>
+        <p class="desktop-project-description-3">
+          A daily selection of privately <br> 
+          personalized reads, no accounts or <br> 
+          sign-ups required has been <br> 
+          the industry's standard.
+        </p>
+        <ul class="project-list">
+          <li class="project-item">
+            html
+          </li>
+          <li class="project-item">
+            bootstrap
+          </li>
+          <li class="project-item">
+            Ruby
+          </li>
+        </ul>
+      </div>
```

Comment:
When I hover on a work, it should show the "Show Project" button with the design described in Figma as described above 😀